<launch>
     <!-- Distances to hydrophone 1 -->
	<param name="hydrophones_dx" value="0.1"/>
	<param name="hydrophones_dy" value="0.1"/>
	<param name="hydrophones_dz" value="-0.1"/>
	<!-- Frequency of each pinger -->
	<param name="pinger_frequency_1" value="25000"/>
	<param name="pinger_frequency_2" value="30000"/>
	<param name="pinger_frequency_3" value="35000"/>
	<param name="pinger_frequency_4" value="40000"/>

	<param name="hydrophones_time_unit" value="0.0000001"/>

	<node name="hydrophones_bearing"
		pkg="state_estimation"
		type="hydrophones_bearing.py"
		output="screen"
		respawn="true">
	</node>
</launch>
